torch_glow/tests/nodes/clamp_test.py

```diff
@@ -0,0 +1,15 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import torch
+
+from tests.utils import jitVsGlow
+
+
+def test_clamp():
+    """Test of the PyTorch min Node on Glow."""
+
+    def test_f(x):
+        return x * torch.clamp(x + 3.0, 0.0, 6.0) * (1.0 / 6.0)
```

Comment:
what's the point of the ` * (1.0 / 6.0)`?

Reply:
Oh lol, sorry, I wanted this for an implementation of hswish I was working on in another place, and just copied that over as a unit test.  Let me change it.